### DISCUSSION TITLE
What is the default port version if none specified

### DISCUSSION THREAD

**Discussion by kcrif** [microsoft/vcpkg#39282] [Q&A] [answered]
Hi,
should a port version always be specified, or what is the default port version when working with manifest mode?

Which port version from zlib version will be used if none specified 

```
  "overrides": [
    { "name": "zlib", "version": "1.2.11" },
  ]
```

**Comment by Osyotr** [comment, marked as answer]
Default is `#0`, so if you want something newer you should specify port-version manually.
